examples/coordinates/plot_sgr-coordinate-frame.py

```diff
@@ -205,13 +171,15 @@ def sgr_to_galactic(sgr_coord, gal_frame):
 ##############################################################################
 # Plot the points in both coordinate systems:
 
-fig,axes = plt.subplots(2, 1, figsize=(6,12),
-                        subplot_kw={'projection': 'aitoff'})
+fig, axes = plt.subplots(2, 1, figsize=(8, 10),
```

Comment:
I think not, because the whole point of the example is to demonstrate the use of a custom coordinate system, which `WCSAxes` I believe does *not* support (because it's not in the FITS-WCS standard).   And to try to get it to would require a lot more code that I think would distract from the point of this example?

But it's probably a good idea to have an example of how you *might* do this...